chat-plugins/mafia.js

```diff
@@ -229,10 +228,10 @@ class MafiaTracker extends Rooms.RoomGame {
 				}
 			}
 			// Determine the role's alignment
-			for (let j = 0; j < target.length; j++) {
-				let id = toId(target[j]);
+			for (let [j, targetId] of target.entries()) {
+				let id = toId(targetId);
 				if (MafiaData.alignments[id]) {
-					if (typeof MafiaData.alignments[id] === 'string') target[j] = MafiaData.alignments[id];
+					if (typeof MafiaData.alignments[id] === 'string') targetId = MafiaData.alignments[id];
```

Comment:
Does this actually do what you think it does?

Reply:
yes

Reply:
...what do you think it does?

As far as I can tell, `targetId` is never read, except in line 232 right after it's declared.

Reply:
Poke

Reply:
I misinterpreted what you asked, whoops, will fix

Reply:
fixed this and the PR is updated